pkgs/development/python-modules/pijuice/default.nix

```
@@ -0,0 +1,41 @@
+{ lib, fetchFromGitHub, buildPythonPackage, pythonOlder,
+smbus-cffi, urwid }:
+
+buildPythonPackage rec {
+  pname = "pijuice";
+  version = "1.7";
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "PiSupply";
+    repo = "PiJuice";
+    # Unfortunately this project doesn't tag releases. This commit altered the
+    # __version__ field in pijuice.py
```

I think this is a better writing for this particular case:

```suggestion
    # rev hash retrieved from the latest modification on file Software/Source/VERSION, as this project
    # does not use Github tags facility
```

(Yes, there is a VERSION file with the same commit hash)